pkg/download/download.go

```diff
@@ -79,6 +80,12 @@ func download(client *http.Client, u *url.URL, path string, extract bool) error
 		}
 		defer resp.Body.Close()
 
+		if resp.StatusCode == http.StatusNotFound {
+			return backoff.Permanent(errors.New("url returned not found"))
```

Comment:
For now it sounds like we should error here to ensure retries unless we host the kinetic feed.

Reply:
Just dropped this out so we're now retrying everything.